src/components/q-and-a/nav.tsx

```diff
@@ -0,0 +1,28 @@
+import { QAndAButton } from "./buttons";
+import { qAndAContent } from "./constants";
+
+type Props = {
+  activeQuestionI: number;
+  itemClick: (index: number) => void;
+};
+
+export function QANav({ activeQuestionI, itemClick }: Props) {
+  return (
+    <nav className="lg:h-[calc(100vh-124px)] h-full lg:overflow-auto overflow-hidden">
```

Comment:
nit: Can you tell me where the `124px` is coming from? Maybe we can put it in a **const** so that it's obvious.